NEWS.md

```diff
@@ -7,42 +7,45 @@
 - ([#16213](https://github.com/rstudio/rstudio/issues/16213)): Improves legibility of highlighted code when RStudio debugger is active
 - ([#15945](https://github.com/rstudio/rstudio/issues/15945)): Adds a user preference to disable showing the splash screen at startup
 - ([#15614](https://github.com/rstudio/rstudio/issues/15614)): The splash screen closes when clicked with the mouse
 - ([#16111](https://github.com/rstudio/rstudio/issues/16111)): Increases the default console buffer size to 10000 lines
-- ([#11661](https://github.com/rstudio/rstudio/issues/11661)): The Zoom button in the RStudio Desktop plots pane now brings an existing zoomed plot window to the foreground 
+- ([#11661](https://github.com/rstudio/rstudio/issues/11661)): The Zoom button in the RStudio Desktop plots pane now brings an existing zoomed plot window to the foreground
 - ([#16009](https://github.com/rstudio/rstudio/issues/16009)): When manually checking for updates (after previously ignoring an update), add an option to stop ignoring updates prompts
 - ([#15988](https://github.com/rstudio/rstudio/issues/15988)): Posit Product Documentation theme v7.0.0; adds cookie consent, several style updates, accessibility fixes, dark theme improvements
 - ([#16214](https://github.com/rstudio/rstudio/issues/16214)): Introduces a new structure for release note entries
 - ([#16226](https://github.com/rstudio/rstudio/issues/16226)): Added option (off by default) to zoom RStudio Desktop UI with Ctrl/Cmd+mouse wheel
 
 #### Posit Workbench
 
+- (rstudio-pro#8093): Workbench homepage has been redesigned to provide a more efficient and intuitive user experience, with a focus on projects
+- (rstudio-pro#6768): Initial support for displaying Positron and VS Code projects on the homepage alongside RStudio projects
 - (rstudio-pro#8162): RStudio now supports the display of a custom "Metadata" column in the Packages pane, for sessions using [Posit Package Manager](https://posit.co/products/enterprise/package-manager/)
 - ([#14083](https://github.com/rstudio/rstudio/issues/14083)): Allows for custom certificate bundles for GitHub Copilot
 - (rstudio-pro#5357): Allows strict enforcement of the user limit specified by the Posit Workbench product license
 - (rstudio-pro#7533): Posit Product Documentation theme v7.0.1; adds cookie consent, several style updates, accessibility fixes, dark theme improvements
 - (rstudio-pro#8186): Enables Positron Pro sessions by default
 - (rstudio-pro#7599): Backs up existing Positron Pro configuration files and installs defaults during upgrade
-- (rstudio-pro##8697) : Adds a product check request that calls to a Posit server. This is controlled by the `server-product-check` configuration setting in `rserver.conf`, default `1`.
+- (rstudio-pro#8697): Adds a product check request that calls to a Posit server. This is controlled by the `server-product-check` configuration setting in `rserver.conf`, default `1`.
```

Comment:
Can we change this to be almost verbatim with Connect? https://docs.posit.co/connect/news/#posit-connect-2025.06.0


> Workbench runs a product check request that notifies a Posit server when the service is set up and actively using a license. This check helps us proactively support customers by confirming successful installation. This is controlled by the `server-product-check` configuration setting, enabled by default (`1`). (#31606)